includes/class-boldgrid-backup-activator.php

```diff
@@ -90,7 +90,31 @@ public static function activate() {
 	 * @return bool
 	 */
 	public static function on_post_activate() {
-		return ! self::$just_activated && '1' === get_option( self::$option );
+		/*
```

Comment:
Does all this logic need to live here? Can we simply change it to:

```
// On Plugin activation, ensure our auto_update settings are synced with WordPress'.
$core = apply_filters( 'boldgrid_backup_get_core', null );
$core->auto_updates->sync_from_wordpress();
```

Reply:
Yes. The logic needs to be there. This should only run during plugin activation, and only on versions of wordpress 5.5 and newer. 

I tried it without all the fancy logic, and that's why you ended up with the bug you had when you would save the settings in TU, then go back to the settings page, and the settings didn't save. I had thought this method only fired directly after plugin activation, however, it would appear that it fires other times as well.

Reply:
Sorry, I didn't mean logic - I meant code.

The list of things to do on plugin activation could grow over time. We could make this method easier to maintain by adding 3 lines instead of 22.